app/assets/stylesheets/to_change_everything/_common.scss

```diff
@@ -12,1062 +12,1072 @@
 
 /*------------------MORE TEXTS LINKS--------------------------*/
 
+/*------------------FOOTER----------------------------------*/
+
+/*------------------RESPONSIVE VIDEO EMBED------------------*/
```

Comment:
Are all of the changes in this file related to TCE SVG icons?
Or is there a rebase missing?
Can we move them to a separate PR if they're not part of the TCE SVG icons change, please?

Reply:
Ooooh. Is this just a indentation/linting diff?
If so, for PR review readability and clarity and stuff, would you mind breaking all of the linter/style changes out into a separate PR, please?

Then this PR can be really narrowly scoped and focused.

Reply:
I reverted the linting changes. I believe _common.scss still indicates changes because of the comments here, don't mind this file, there aren't any changes relevant in it.
I will create another PR for linting changes after this has been merged to avoid conflicts.